de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/AbstractIncomingReferencePointNegotiation.java

```diff
@@ -64,11 +65,11 @@
     /** used to handle file transmissions **/
     protected TransferListener transferListener = null;
 
-    public AbstractIncomingProjectNegotiation(
+    public AbstractIncomingReferencePointNegotiation(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 12 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWaWrfcIsSPV6ZXI_B9J&open=AWaWrfcIsSPV6ZXI_B9J)

[ISSUE_KEY]: AWaWrfcIsSPV6ZXI_B9J